.circleci/config.yml

```diff
@@ -179,6 +179,40 @@ jobs:
       - EMTEST_DETECT_TEMPFILE_LEAKS=0
       - DISPLAY=:0
       - GALLIUM_DRIVER=softpipe # TODO: use the default llvmpipe when it supports more extensions
+  test-browser-chrome:
+    <<: *defaults
+    environment:
+      - EMTEST_LACKS_SOUND_HARDWARE=1
+      - EMTEST_DETECT_TEMPFILE_LEAKS=0
+    working_directory: ~/
+    steps:
+      - checkout:
+          path: emscripten/
+      - attach_workspace:
+          # Must be absolute path or relative path from working_directory
+          at: ~/
+      - run:
+          name: install package dependencies
+          command: |
+            apt-get update -q
+            # install chromium-browser in order to ensure we have all the
+            # dependecies for chrome.
+            apt-get install -q -y unzip xvfb chromium-browser
```

Comment:
Kind of surprising that there's no way to install just the dependencies of a package. But I don't see one either.